test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -748,6 +748,15 @@ func unrollWorkloadTemplate(b *testing.B, wt []op, w *workload) []op {
 }
 
 func runWorkload(ctx context.Context, b *testing.B, tc *testCase, w *workload) []DataItem {
+	start := time.Now()
+	defer func() {
+		duration := time.Now().Sub(start)
+		// This includes startup and shutdown time and thus does not
```

Comment:
What do you mean the shutdown time, including cleanup? Currently it doesn't, if we want,we should use b.Cleanup()

Reply:
> To make sure, this will not impact the perf-test dashboard, right?

The set of workloads changes, so there is some impact. For example, the new Scheduling*AffinityWithNSSelector tests with 500 nodes will get added to the dashboard.

You are right, any additional cleanup will not be included in the duration. Let me change that...

Reply:
Fixed by invoking the function via `b.Cleanup`.